.github/workflows/kind-e2e.yaml

```diff
@@ -43,6 +44,8 @@ jobs:
           kind-version: v0.9.0
           kind-image-sha: sha256:98cf5288864662e37115e362b23e4369c8c4a408f99cbc06e58ac30ddc721600
           kingress: contour
+        - test-suite: ./test/conformance/api/v1alpha1
```

Comment:
nit: I'd put some space between the two blocks, since they act on different dimensions of the matrix

Reply:
👍️ done.